cluster/juju/layers/kubernetes-worker/layer.yaml

```diff
@@ -28,3 +28,4 @@ options:
     server_key_path: '/root/cdk/server.key'
     client_certificate_path: '/root/cdk/client.crt'
```

Comment:
are we keeping the client keys for any specific purpose? or is this for compat reasons since we have charms out there in the wild with client keys?